lib/ex_docker_build/cli.ex

```diff
@@ -0,0 +1,15 @@
+defmodule ExDockerBuild.CLI do
+  def main([]) do
+    IO.puts("You forgot to pass the path to the Dockerfile")
+  end
+
+  def main(args) do
```

Comment:
args is a list of all arguments passed to the escript, i think you should pattern match on that to extract the path `[path]` because if for some reason the user makes a typo, there will be an error e.g `./ex_docker_build path/to/file other_arg`